programs/svm-spoke/src/instructions/admin.rs

```diff
@@ -247,4 +250,39 @@ pub fn relay_root_bundle(
     Ok(())
 }
 
-// TODO: add emergency_delete_root_bundle
+#[event_cpi]
+#[derive(Accounts)]
+#[instruction(root_bundle_id: u32)]
+pub struct EmergencyDeleteRootBundle<'info> {
+    #[account(
+        mut,
+        constraint = is_local_or_remote_owner(&signer, &state) @ CustomError::NotOwner
+    )]
+    pub signer: Signer<'info>,
+
+    // TODO: standardize usage of state.seed vs state.key()
+    #[account(mut, seeds = [b"state", state.seed.to_le_bytes().as_ref()], bump)]
+    pub state: Account<'info, State>,
+
+    // TODO: consider deriving seed from state.seed instead of state.key() as this could be cheaper (need to verify).
+    #[account(mut,
+        seeds =[b"root_bundle", state.key().as_ref(), root_bundle_id.to_le_bytes().as_ref()],
+        close = signer,
```

Comment:
Not sure how this work for remote calls as the signer would be PDA. Have to add tests and see if PDA would hold returned lamports.

Reply:
I will modify RelayRootBundle to include dedicated payer as CCTP requires this (same as in SetEnableRoute), so I will change account close recipient to the same payer when I add CCTP support for these calls